cvat/apps/dataset_manager/formats/yolo.py

```diff
@@ -44,11 +59,117 @@ def _import(src_file, temp_dir, instance_data, load_data_callback=None, **kwargs
         except Exception: # nosec
             pass
         if frame_info is not None:
-            image_info[frame] = (frame_info['height'], frame_info['width'])
+            image_info[frame] = (frame_info["height"], frame_info["width"])
 
-    detect_dataset(temp_dir, format_name='yolo', importer=dm_env.importers.get('yolo'))
-    dataset = Dataset.import_from(temp_dir, 'yolo',
-        env=dm_env, image_info=image_info)
+    detect_dataset(temp_dir, format_name=format_name, importer=dm_env.importers.get(format_name))
+    dataset = Dataset.import_from(temp_dir, format_name,
+        env=dm_env, image_info=image_info, **(import_kwargs or {}))
     if load_data_callback is not None:
         load_data_callback(dataset, instance_data)
     import_dm_annotations(dataset, instance_data)
+
+
+@importer(name='YOLO', ext='ZIP', version='1.1')
+def _import(src_file, temp_dir, instance_data, load_data_callback=None, **kwargs):
+    _import_common(
+        src_file,
+        temp_dir,
+        instance_data,
+        load_data_callback=load_data_callback,
+        **kwargs
+    )
+
+
+@exporter(name='YOLOv8 Detection', ext='ZIP', version='1.0')
+def _export(dst_file, temp_dir, instance_data, save_images=False):
+    with GetCVATDataExtractor(instance_data, include_images=save_images) as extractor:
+        dataset = Dataset.from_extractors(extractor, env=dm_env)
+        dataset.export(temp_dir, 'yolov8', save_images=save_images)
+    make_zip_archive(temp_dir, dst_file)
+
+
+@exporter(name='YOLOv8 Oriented Bounding Boxes', ext='ZIP', version='1.0')
+def _export(dst_file, temp_dir, instance_data, save_images=False):
+    with GetCVATDataExtractor(instance_data, include_images=save_images) as extractor:
+        dataset = Dataset.from_extractors(extractor, env=dm_env)
+        dataset.export(temp_dir, 'yolov8_oriented_boxes', save_images=save_images)
+    make_zip_archive(temp_dir, dst_file)
+
+
+@exporter(name='YOLOv8 Segmentation', ext='ZIP', version='1.0')
+def _export(dst_file, temp_dir, instance_data, save_images=False):
+    with GetCVATDataExtractor(instance_data, include_images=save_images) as extractor:
+        dataset = Dataset.from_extractors(extractor, env=dm_env)
+        dataset.export(temp_dir, 'yolov8_segmentation', save_images=save_images)
+    make_zip_archive(temp_dir, dst_file)
+
+
+@exporter(name='YOLOv8 Pose', ext='ZIP', version='1.0')
+def _export(dst_file, temp_dir, instance_data, save_images=False):
+    with GetCVATDataExtractor(instance_data, include_images=save_images) as extractor:
+        dataset = Dataset.from_extractors(extractor, env=dm_env)
+        dataset.export(temp_dir, 'yolov8_pose', save_images=save_images)
+    make_zip_archive(temp_dir, dst_file)
+
+
+@importer(name='YOLOv8 Detection', ext="ZIP", version="1.0")
+def _import(src_file, temp_dir, instance_data, load_data_callback=None, **kwargs):
```

Comment:
I think you can shorten these functions significantly by removing all named parameters. You're just passing them to `_import_common` unchanged; `**kwargs` already accomplishes that.

Reply:
done

Reply:
Not quite; I meant something along the lines of:

```python
@importer(name='YOLOv8 Detection', ext="ZIP", version="1.0")
def _import_yolov8_detection(*args, **kwargs):
    _import_common(*args, format_name="yolov8", **kwargs)
```

Reply:
ahh, done